### PR TITLE
Created new record

### PR DESCRIPTION
**Pull request by antobrita**

### CODE REVIEW

File: new/PRS13297Zamalakot.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ዘመለኮት</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Zamalakot</persName>
+                    <faith type="EOTC"/>
+                    <floruit notBefore="1600" notAfter="1800"></floruit>
+
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS13297Zamalakot" passive="GCA001">
+                    <desc>He is mentioned in the colophon of the manuscript GCA001 as commissioner of the manuscript and donor to the church of Faqādā</desc>
```
Comment (MassimoVilla) [inline new/PRS13297Zamalakot.xml:56]: In the desc, GCA001 can be marked up this way: `<ref type="mss" corresp="GCA001"/>`
If Faqādā has its own ID, also this word can be marked up as a placeName: `<placeName ref="..."/>`

Comment (antobrita) [reply new/PRS13297Zamalakot.xml:56]: > In the desc, GCA001 can be marked up this way: `<ref type="mss" corresp="GCA001"/>`

Alright! Thank you, Massimo

> If Faqādā has its own ID, also this word can be marked up as a placeName: `<placeName ref="..."/>`

There are two placeName called Faqādā in BM, recorded by EthioSpare. I'm checking in the single manuscripts preserved in these monasteries if some of the names found in our ms are attested also there. If not, I will create a new record for the place Faqādā. I will come back to you when I'm done with the search.

Comment (antobrita) [reply new/PRS13297Zamalakot.xml:56]: Sorry, I have to correct what I wrote before. There is only one church in BM (Faqādā Māryām), the other is a location. Anyway, I found in three mss of Faqādā Māryām (FBM-001, FBM-003, FBM-016) the name Baʾǝda Māryām attested as owner. In our ms (GCA001) is also present in the subscriptiones and blessing formulae a Baʾǝda Māryām and the ms. was donated to a መቅደሰ፡ ፈቃዳ፡. Do you think our  መቅደሰ፡ ፈቃዳ፡ can be identified with Faqādā Māryām recorded by Ethiospare?

Comment (SusanneHummel) [reply new/PRS13297Zamalakot.xml:56]: Dear Anto, as far as I remember correctly, this was exactly what I also assumed when preparing our inventory. We also stated Faqādā as origin/provenance in our inventory. I have to check in the office tomoroow wehther if I have more details on it.

Comment (antobrita) [reply new/PRS13297Zamalakot.xml:56]: Thank you, Sanne, for that. I remembered something about a known place recurring also in our manuscripts but I did not linked to Faqādā. Alright, I will wait that you check before I proceed with creating a new record both for Faqādā and Baʾǝda Māryām.

P.S. Tomorrow morning we have our breakfast at 9.30. Massimo, if you want to join us you are more than welcome!

Comment (SusanneHummel) [reply new/PRS13297Zamalakot.xml:56]: You could add also here that this person has a relation to Faqādā Māryām


```
<affiliation>
<placeName ref="INS0071FBM">Faqādā Māryām</placeName>
</affiliation>
```




I have never done this but checked in the TEI guidelines: https://tei-c.org/release/doc/tei-p5-doc/de/html/ref-affiliation.html

**Review by SusanneHummel — APPROVED** [review]

**Comment by antobrita** [comment]
I made the change requested by Massimo and added the relation to the place, as explained in the guidelines https://betamasaheft.eu/Guidelines/?q=saws&id=relations (Person to Places)

**Comment by antobrita** [comment]
as requested by Sanne.
(sorry I sent my previous message without the final part)

**Review by MassimoVilla — APPROVED** [review]

**Comment by SusanneHummel** [comment]
It looks perfect! I will merge it and keep it for me as a Vorlage for the place relation

**Pull request merged**